### ISSUE TITLE
UX/UI: Encourage mint selection when default mint is not allowing minting of new tokens

### ISSUE THREAD

**Issue by swedishfrenchpress** [cashubtc/eNuts#342]
We got some good feedback from @gbks.

**Is your feature request related to a problem? Please describe.**
![enuts-mint-down](https://github.com/cashubtc/eNuts/assets/78821053/47299bb8-1ce1-4e5f-b1f2-f754b1afbce8)
The application recommends connecting to the eNuts mint that is currently unavailable, allowing users to attempt an action that leads to an error. To me [Erik] The error message  is unclear. Is the mint down to a technical error? Or is at a capacity issue?

**A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]**
I'm frustrated when I am allowed to choose minting options that are not available and enter amounts, only to be stopped by an error message stating that the mint is at capacity.

**Describe the solution you'd like**
A pre-check that verifies whether minting is possible on the default suggested mint before the user is allowed to add the mint and given a success message. If minting is at capacity or unavailable this check should prevent the user from proceeding and being prevented from minting down the line.

**Describe alternatives you've considered**
As an alternative to the pre-check, if the mint is at capacity, adjust the error messaging to clarify that there is no error but rather a capacity limitation. This message could be more neutral and provide options for alternative actions, such as adding another mint.

**Additional context**
Users who encounter this problem during their first touch point with eNuts may find this frustrating and the error messages confusing, which might turn them away from using the app.

I'll let @gbks chime in with any additional insights or suggestions. I'm including mine below to get the ball rolling. 

- I think it might be better to skip mentioning that the default mint is has an error is at capacity. The user doesn't need to know the reason why the mint is down or why it can't mint tokens. Instead, we could use just mention that the mint is currently unavailable and use more positive language to encourage users to choose a mint from https://bitcoinmints.com/. This keeps the tone positive and avoids making the user's first experience feel like an error or suggesting there's a problem with the app.
- If we want to allow the user to add the default mint anyway, we could just update the error message copy. Perhaps something like this the second example.
![Frame 5916](https://github.com/cashubtc/eNuts/assets/78821053/4ed6c9ad-1b9c-41e7-9f5e-052f086a83a7)

**Comment by GBKS** [comment]
Thanks for summarizing my feedback here, very thorough.

To the last screen on the right, it still looks like the mint is down, which makes me doubt their reliability and I may not trust them in the future. Still think that this is different from the mint being run perfectly, but just being at capacity. Maybe there need to be 2 different error messages there.

**Comment by swedishfrenchpress** [comment]
> Thanks for summarizing my feedback here, very thorough.
> 
> To the last screen on the right, it still looks like the mint is down, which makes me doubt their reliability and I may not trust them in the future. Still think that this is different from the mint being run perfectly, but just being at capacity (if that is actually a thing). Maybe there need to be 2 different error messages there.

Another Cashu dev mentioned that the eNuts beta mint is unavailable because it's in 'shutdown mode,' and advised me to try a different mint. What states can mints be in? Can apps query these states? If possible, we should customize the error messages to explain the situation in layman's terms and suggest alternative actions.

**Comment by GBKS** [comment]
> If possible, we should customize the error messages to explain the situation in layman's terms and suggest alternative actions.

1000000% yes. If we are trusting mints with our bitcoin, then we need to be super-well informed with what is going on with them. And mints need to be as transparent as possible.
